### PR TITLE
[Fix-5509]: createSchedule interface, process definition id -> process definition code

### PR DESCRIPTION
**Pull request by wen-hemin**
close: #5509 

from #5498

### CODE REVIEW

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/SchedulerController.java [hunk 1]
```diff
@@ -108,14 +108,14 @@
     @AccessLogAnnotation(ignoreRequestArgs = "loginUser")
     public Result createSchedule(@ApiIgnore @RequestAttribute(value = SESSION_USER) User loginUser,
                                  @ApiParam(name = "projectName", value = "PROJECT_NAME", required = true) @PathVariable String projectName,
```
Comment (brave-lee) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/SchedulerController.java]: Please replace the projectName with projectCode

**Comment by brave-lee** [comment]
Please fix ut

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/SchedulerController.java (hunk 1 above)
Comment (wen-hemin) [reply dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/SchedulerController.java]: Thx, i will fix

Comment (wen-hemin) [reply dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/SchedulerController.java]: projectCode replaces projectName in Other interface, submit in other PR later

**Review by brave-lee — COMMENTED** [review]
+1

**Pull request merged**